src/AngleSharp.Css/Values/Functions/CssRunningValue.cs

```diff
@@ -59,7 +59,7 @@ public CssRunningValue(String ident)
         /// </summary>
         /// <param name="other">The value to check against.</param>
         /// <returns>True if both are equal, otherwise false.</returns>
-        public Boolean Equals(CssRunningValue other) => _ident.Equals(other._ident);
+        public Boolean Equals(CssRunningValue other) => other is not null && _ident == other._ident;
```

Comment:
For this scenario (single primitive string field) we not have 3 different constructs (e.g., [CssAttrValue.cs](https://github.com/AngleSharp/AngleSharp.Css/pull/163/files#diff-30b1230b47b07604c15130f3f2ac5a6a275a934ea26f4fa50ddea61500c9523c)). Intentional?

Reply:
I've fixed CssFontFormatValue to be single line

Reply:
And also `CssAttrValue`